Basic Rules.txt

```diff
@@ -350,6 +350,47 @@ Using a medikit:
 Medic-kit, non-medic user: works as if you are a medic with skill 30.
 Medic-kit, medic user: Works as normal, but add 30 to amount healed.
 
+
+==============================
+Death, Dying and Downed
+==============================
+Oh no! You have less than 1 health point left! What now!?!
+
+The good news is you are not* dead yet. You are bleeding out. 
+When your character falls below 1 hit point, you get 3 saving throws called Death 
+Saving Throws. These throws are not tied to any attribute. When you fail any three 
+times, you die. If you succeed, you do not lose one of these three Death 
+Saving Throws, but will still have to roll next round if you are not stabalized. 
+Once you have expended a saving throw, the next time you fall 
+below 1 health point your saving throws fall to 2. the next time, you have 1. you 
+can only replenish these saving throws by going a whole encounter without having 
+to use any. Each encounter that you go without expending a saving throw 
+replenishes one saving throw up to a maximum of three.
+

```

Comment:
I redid the paragraph because I thought it was unclear on multiple points. It seemed a little unclear to me whether the player would be able to regain a saving throw by not *losing* one for a given encounter or *using* one for an encounter. I chose to reword it as *failing* a saving throw, as a slight buff. It will be quite a bit easier to die, even easier if you have to not be downed for an entire encounter to regain one. I also abbreviated "Death Saving Throws" and used it profusely, to try and make the writing completely unambiguous to the player (that we aren't talking about, "saving throws" in general):

>The good news is you are not* dead yet. But you are doing what's known as "bleeding out". 
The first time your character falls below 1 hit point, you get 3 saving throws called Death 
Saving Throws (or 'DSTs' for short). You roll one saving throw per turn that you are downed 
and these DSTs are not modified by any attribute. If you succeed a DST, (see 'Downed', below) 
you continue bleeding out; however, you get to roll again next turn. If you fail 3 DSTs, you die. 
	Once you fail a DST, the next time you fall below 1 health point you only get 2 DSTs 
(2 failed throws and you die). If you failed a DST during the first two times you were downed 
and you are downed a *third* time, you only get 1 DST. 
	You regain a DST by going a whole encounter without failing a DST. You can have a maximum 
of 3 saving throws at any one time.

Reply:
done